integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java

```diff
@@ -144,7 +144,12 @@ public synchronized void startServerUsingCarbonHome(String carbonHome, Map<Strin
                 }
             }));
 
-            managementPort = 9154 + portOffset;
+            if (commandMap.get("managementPort") != null) {
+                managementPort = Integer.parseInt(commandMap.get("managementPort")) + portOffset;
+            } else {
+                managementPort = 9763 + portOffset;
```

Comment:
shall we make MI port as default ( 9154 ) .. then we don't need to edit MI , automation xml files